rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -2925,6 +2925,158 @@ public void addLockWhitelistAddress_invalidAddress() throws IOException {
+    }
+
+    @Test(expected = ArithmeticException.class)
+    public void setLockWhitelistDisableBlockDelay_disableBlockDelayBIBiggerThanInt() throws IOException, BlockStoreException {
+        Transaction mockedTx = mock(Transaction.class);
+        byte[] senderBytes = ECKey.fromPublicOnly(Hex.decode(
+            // Public key hex of the authorized whitelist admin in regtest, taken from BridgeRegTestConstants
+            "04641fb250d7ca7a1cb4f530588e978013038ec4294d084d248869dd54d98873e45c61d00ceeaeeb9e35eab19fa5fbd8f07cb8a5f0ddba26b4d4b18349c09199ad"
+        )).getAddress();
+        RskAddress sender = new RskAddress(senderBytes);
+        when(mockedTx.getSender()).thenReturn(sender);
+        LockWhitelist mockedWhitelist = mock(LockWhitelist.class);
+        when(mockedWhitelist.isDisableBlockSet()).thenReturn(false);
+        int bestChainHeight = 10;
+        BtcBlockstoreWithCache btcBlockStore = mock(BtcBlockstoreWithCache.class);
+        StoredBlock storedBlock = mock(StoredBlock.class);
+        when(storedBlock.getHeight()).thenReturn(bestChainHeight);
+        when(btcBlockStore.getChainHead()).thenReturn(storedBlock);
+        BridgeSupport bridgeSupport = getBridgeSupportWithMocksAndBtcBlockstoreForWhitelistTests(mockedWhitelist,btcBlockStore);
+        //Duplicate Int Max Value by 2 because its signed and add 1 to pass the limit
+        BigInteger disableBlockDelayBI = BigInteger.valueOf((long)Integer.MAX_VALUE * 2 + 1);
+
+        bridgeSupport.setLockWhitelistDisableBlockDelay(mockedTx, disableBlockDelayBI);
+        verify(mockedWhitelist, never()).put(any(), any());
+    }
+
+    @Test
+    public void setLockWhitelistDisableBlockDelay_overflow() throws IOException, BlockStoreException {
+        Transaction mockedTx = mock(Transaction.class);
+        byte[] senderBytes = ECKey.fromPublicOnly(Hex.decode(
+            // Public key hex of the authorized whitelist admin in regtest, taken from BridgeRegTestConstants
+            "04641fb250d7ca7a1cb4f530588e978013038ec4294d084d248869dd54d98873e45c61d00ceeaeeb9e35eab19fa5fbd8f07cb8a5f0ddba26b4d4b18349c09199ad"
+        )).getAddress();
+        RskAddress sender = new RskAddress(senderBytes);
+        when(mockedTx.getSender()).thenReturn(sender);
+        LockWhitelist mockedWhitelist = mock(LockWhitelist.class);
+        when(mockedWhitelist.isDisableBlockSet()).thenReturn(false);
+        int bestChainHeight = (Integer.MAX_VALUE / 2) + 2;
+        BtcBlockstoreWithCache btcBlockStore = mock(BtcBlockstoreWithCache.class);
+        StoredBlock storedBlock = mock(StoredBlock.class);
+        when(storedBlock.getHeight()).thenReturn(bestChainHeight);
+        when(btcBlockStore.getChainHead()).thenReturn(storedBlock);
+        BridgeSupport bridgeSupport = getBridgeSupportWithMocksAndBtcBlockstoreForWhitelistTests(mockedWhitelist,btcBlockStore);
+
+        BigInteger disableBlockDelayBI = BigInteger.valueOf(Integer.MAX_VALUE / 2);
```

Comment:
If you want to test the case when current block height + delay is negative then use `Integer.MAX_VALUE`  otherwise it won't be negative until the Bitcoin block height is `Integer.MAX_VALUE / 2 + 1` and that will be in 40,000 years.